appdaemontestframework/given_that.py

```diff
@@ -62,18 +61,18 @@ def format_time(timestamp: datetime):
                 else:
                     return state['attributes'].get(attribute)
 
-        self.hass_functions['get_state'].side_effect = get_state_mock
+        self._hass_mocks.hass_functions['get_state'].side_effect = get_state_mock
 
         def entity_exists_mock(entity_id):
             if entity_id in self.mocked_states:
                 return True
             else:
                 return False
```

Comment:
🤣 Who did this ? 😂🤣
(It's very likely that it's me hahaha)

It can be replaced with:
```python
def entity_exists_mock(entity_id):
    return entity_id in self.mocked_states
```

I know it's not related to the PR, but I just saw it and it's a tiny change :)

Reply:
haha, we all do this from time to time. No prob, I changed it.

Reply:
Hahah yes, we all do 😁 Thx for updating